NuGet.Config

```diff
@@ -5,6 +5,13 @@
     <clear />
     <add key="project" value="libs" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="bdn-nightly" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
   </packageSources>
+  <packageSourceMapping>
+    <packageSource key="project">
+      <package pattern="*" />
+    </packageSource>
+    <packageSource key="api.nuget.org">
+      <package pattern="*" />
```

Comment:
If you have stars here, does it do anything? As I understand, this is to specify the package origin on a per dependency basis?

Reply:
it is a name pattern a.k.a. 'what package can be downloaded from that source'